docs/reference/pyboard.py.rst

```diff
@@ -48,6 +52,10 @@ Running ``pyboard.py --help`` gives the following output:
                             available
       --follow              follow the output after running the scripts
                             [default if no scripts given]
+      --no-soft-reset       Prevent performing a soft reset when connecting to MCU
+      --hard-reset          pulse the MCU reset pin to hard-reset the MCU (if your
+                            serial connection wires RTS to reset pin properly)
+      --no-exclusive        Open the serial device shared (not exclusive) access
```

Comment:
It should be detailed that this is a POSIX only function (https://pyserial.readthedocs.io/en/latest/pyserial_api.html) so won't work on windows.

Reply:
not sure what you mean? The code already has a custom setup for Windows, and it already uses those pins properly in there.  I've got no idea why someone deliberately made this work for ESP32/ESP8266 for windows only.  My fix makes it work for linux and WSL too.

Reply:
Here I'm referring to the `exclusive` setting in pyserial. `--no-exclusive` appears to be not possible on windows.